kpi/core/title.adoc

```diff
@@ -6,67 +6,50 @@
 
 .Good quality title implementation rules
 |===
 |Rule |Score
 
-|title is present
+|The title is present
 |1
 
-|title has 3 words or more
+|The title has 3 words or more
 |1
 
-|title has 150 characters or less
+|The title has 150 characters or less
 |1
 
-|title only has printable characters (numbers and letters)
+|The title has only printable characters (numbers and letters)
 |1
 
-|words in title are represented in "Title Case"
+|The words in title are represented in "Title Case"
 |1
 
-|title contains less than 3 acronyms (words with all upper case)
+|The title contains less than 3 acronyms (words with all upper case)
 |1
 
-a|title does not contain bulletin header (regular expression: `[A-Z]{4}\d{2}[\s_]*[A-Z]{4}`)
+a|The title _does not_ contain bulletin header (regular expression: `[A-Z]{4}\d{2}[\s_]*[A-Z]{4}`)
 |1
 
-|title passes a basic spellcheck
+|The title passes a basic spellcheck
 |1
 |===
 
 *Total possible score: 8 (100%)*
 
 === Guidance to score well on this assessment
 
-In the context of WIS2 Global Discovery Catalogues, the product title and abstract are
-the two most relevant elements in the WCMP metadata record.  These two elements
-are presented to the users in search results as well as the product description
-page, and need to focus on highlighting the product’s key characteristics to
-assist users with relevant product search results.
-
-The title should be as specific about the product as possible. For example, if
-the product only contains one parameter, this can be stated in the title;
-however, if the product contains numerous parameters, then a more general term
-should be used in the title, and the parameters stated elsewhere in the
-metadata record (abstract, keywords, etc.).
-
-For a satellite product offering one main data parameter, the title will
-typically define which parameter is contained in the product, and from which
-instrument or instrument type it originates. For example, "AMSR-2 Sea Surface
-Temperature” or “SLSTR L1B radiances and brightness temperatures”. Please see
-the Guidance on WIS Discovery Metadatafootnote:[https://library.wmo.int/index.php?lvl=notice_display&id=6856#.X2TSLZMzYUF]
-for more information.
+The title should be as specific as possible. For example, if the product only contains one parameter, this can be stated in the title; however, if the product contains numerous parameters, then a more general term should be used in the title, and the parameters stated elsewhere in the metadata record (description, themes, keywords, etc.).
+
+For a satellite product offering one main data parameter, the title will typically define which parameter is contained in the product, and from which instrument or instrument type it originates. For example, "AMSR-2 Sea Surface Temperature” or “SLSTR L1B radiances and brightness temperatures”. Please see the Guidance on WIS Discovery Metadatafootnote:[https://library.wmo.int/index.php?lvl=notice_display&id=6856#.X2TSLZMzYUF] for more information.
```

Comment:
We should remove this reference to the Guidance on WIS Discovery Metadata in the library, because it's for WCMP1.3